vocab/credentials/v2/vocabulary.yml

```diff
@@ -136,7 +136,7 @@ property:
 
   - id: termsOfUse
     label: terms of use
-    domain: cred:VerifiableCredential
+    domain: cred:VerifiableCredential,cred:VerifiablePresentation
     range: odrl:Policy
     comment: |
       If specified, the value of the `termsOfUse` property MUST specify one or more terms of use policies under which the creator issued the credential or presentation. If the recipient (a holder or verifier) is not willing to adhere to the specified terms of use, then they do so on their own responsibility and might incur legal liability if they violate the stated Terms of Use. Each `termsOfUse` MUST specify its type, for example, `IssuerPolicy`, and optionally, its instance `id`. The precise contents of each term of use is determined by the specific `TermsOfUse` type definition.
```

Comment:
```suggestion
      If specified, the value of the optional `termsOfUse` property MUST specify one or more terms of use policies under which the issuer issued the credential or presentation. Each `termsOfUse` value MUST specify its type (for example, `IssuerPolicy`) and MAY specify its instance `id`. The precise content of each term of use is determined by the specific `TermsOfUse` type definition. If the recipient (a holder or verifier) violates the specified terms of use, the responsibility is their own, and such violation may incur legal liability.
```

Reply:
I am personally ok with the change, @TallTed, thanks. But I would prefer a domain expert to give the thumbs up before taking this over. My only slight reservation is that this document is, formally, not a recommendation, so I am not sure whether a _MAY_ is appropriate instead of a "may".

Reply:
(I realize that _MUST_ was already part of the text, b.t.w., so my question is general, and not closely related to your change proposal.)

Reply:
Given that the original text was `MUST ... optionally`, I can't imagine any domain expert having an issue with the `MAY`, though they (and I, on rereading) may prefer `MAY additionally`.

Reply:
Changes here look fine to me (speaking as a domain expert).